STYLE_API_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
-# API reference page template
+# API reference style guide
 
-This template provides the basic structure for creating OpenSearch API documentation. It includes the most important elements that should appear in the documentation and helpful suggestions to help support them. 
+This style guide provides a template with the basic structure for OpenSearch API documentation. It includes the basic headings and helpful suggestions for what information is required in each section.
 
-Depending on the intended purpose of the API, *some sections will be required while others may not be applicable*.
+Depending on the intended purpose of the API, *some sections will be required while others may not be applicable*. For example, only PUT or POST APIs require request and response field descriptions.
```

Comment:
Let's remove the second sentence. It's not accurate. See https://opensearch.org/docs/latest/opensearch/rest-api/index-apis/get-index/.

Reply:
agreed. thanks!